Cargo.toml

```diff
@@ -2,8 +2,20 @@
 name = "tmsu"
 version = "0.1.0"
 authors = ["Paul Ruane <paul.ruane@oniony.com>"]
+edition = "2018"
 
-[dependencies.clap]
-version = "~2.27" 
-default-features = false
-features = [ "suggestions", "wrap_help" ]
+# TODO: try to use fewer features of dependencies when possible
+[dependencies]
+# CLI dependencies
+structopt = "0.3.14"
+clap = "2.33.1"
+termion = "1.5.5"
+pretty_env_logger = "0.4.0"
+
+# Basic lib dependencies
+log = "0.4.8"
+dirs = "2.0.2"
+error-chain = "0.12.2"
```

Comment:
`error-chain` makes it very easy to propagate errors with `?`, avoiding a lot of boilerplate. It is a bit less practical when adding context, or to define proper errors for the API.

I guess we will use a different library at some point. Relevant article: https://blog.yoshuawuyts.com/error-handling-survey/